examples/interactive_play.py

```diff
@@ -356,13 +361,21 @@ def play_env(env, args, config):
         else:
             keys = pygame.key.get_pressed()
 
+        if not args.no_render and keys[pygame.K_x]:
+            if agent_to_control == "AGENT_0":
+                agent_to_control = "AGENT_1"
+            else:
+                agent_to_control = "AGENT_0"
+            print(f"Swapped to {agent_to_control}")
```

Comment:
Do we have anything better than print we could use to display this message?
```suggestion
            print(f"Controlled agent changed. Controlling agent {agent_to_control}.")
```